she_codes_news/news/views.py

```diff
@@ -1,7 +1,19 @@
-from django.views import generic
+import datetime
+from urllib import parse
+from zoneinfo import ZoneInfo
+
+from django.conf import settings
 from django.urls import reverse_lazy
-from .models import NewsStory
+from django.views import generic
+
 from .forms import StoryForm
+from .models import NewsStory
+
+
+def get_tz_name(request):
+    if request.COOKIES.get('timezone'):
```

Comment:
since we've already got `django.utils.timezone` set by the middleware, could get the current tz from that rather than re-parsing the cookie?
```
current_tz = timezone.get_current_timezone()
``` ?

Reply:
yes good point!!